local/local.go

```diff
@@ -683,3 +685,10 @@ func v1Config(inspect types.ImageInspect) (v1.ConfigFile, error) {
 		Config: config,
 	}, nil
 }
+
+func ensureReaderClosed(r io.ReadCloser) error {
+	if _, err := io.Copy(ioutil.Discard, r); err !=nil {
+		return err
+	}
+	return r.Close()
```

Comment:
This is a different behavior in handling the close. If the defer call above errored, we'd just be dropping that on the floor. This seems better, but just wanted to call this change out.

Reply:
good catch! fixed